packages/lsp/src/plugin.ts

```diff
@@ -9,6 +9,10 @@
  * and run jlpm build:schema to regenerate this file.
  */
 
+/**
+ * Enable or disable the language server services.
+ */
+export type Activate = 'off' | 'on';
```

Comment:
Does this need to be an exported type? Can `'off' | 'on'` just be used directly without a named type:
```typescript
activate?: 'off' | 'on';
```

Reply:
This interface is generated automatically from the JSON schema file using `json-schema-to-typescript`